.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 😓 Build break
+about: Use this to report a broken build in jenkins
```

Comment:
Is this a default?

Reply:
No - why do you ask?

Reply:
Ah ... because it's a duplicate of buildbreak.md ... Removed